Lib/test/test_strptime.py

```diff
@@ -521,8 +521,17 @@ def test_date_time_locale(self):
                       'my_MM', 'shn_MM')
     def test_date_time_locale2(self):
         # Test %c directive
+        loc = locale.getlocale(locale.LC_TIME)[0]
+        if sys.platform.startswith(('sunos', 'solaris')):
```

Comment:
This can be safely just 'sunos'. AFAIK, neither Oracle nor OpenSolaris forks will report 'solaris' here.